src/features/klant/klant-details/KlantDetails.vue

```diff
@@ -75,9 +80,24 @@ const emit = defineEmits<{
   load: [data: Klant];
   loading: [data: boolean];
   error: [data: boolean];
+  noData: [];
 }>();
 
-watchEffect(() => klant.value && emit("load", klant.value));
+watchEffect(() => {
+  if (klant.value) {
+    emit("load", klant.value);
+  }
+
+  if (
```

Comment:
het lijkt me netter om dit in een losse watchEffect te doen. Je hebt nu kans dat het load event meerdere keren afgevuurd wordt als de refs waar hieronder op gecheckt wordt veranderen.

Reply:
9f9a14e9dc4fdccae8260da9d928bb7bcb4d5d1c